### PR TITLE
Blog Post: Vitor Pamplona LTS

### PR DESCRIPTION
**Pull request by Arvin21M**
**Build Preview:**

- [/blog/vitor-pamplona-receives-lts-grant
](https://os-website-git-vitor-pamplona-lts-open-sats.vercel.app/blog/vitor-pamplona-receives-lts-grant)
**Social Preview:**

- image

### CODE REVIEW

**Comment by dergigi** [comment]
LOVE the cover image, oh my God @Arvin21M you're good at that 😁💜

File: data/blog/vitor-pamplona-receives-lts-grant.mdx [hunk 1]
```diff
@@ -0,0 +1,69 @@
+---
+title: "Long-Term Support For Vitor Pamplona"
+date: '2024-08-14'
```
Comment (dergigi) [inline data/blog/vitor-pamplona-receives-lts-grant.mdx:3]:
```suggestion
date: '2024-08-19'
```

File: data/blog/vitor-pamplona-receives-lts-grant.mdx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+OpenSats is proud to announce that Vitor Pamplona has been granted long-term
+support for his exceptional contributions to the nostr ecosystem.
+
+[Vitor](https://github.com/vitorpamplona) is the creator of Amethyst, a
+widely-used and configurable nostr client for Android users, and one of the
+architects behind the [new hyper-private DM
+protocol](https://github.com/nostr-protocol/nips/blob/master/17.md) for nostr.
+Among its [extensive features](https://github.com/vitorpamplona/amethyst#),
+Amethyst supports multiple user accounts, live streams, marketplaces, group
+chats, custom feeds via data vending machines, and integrates bitcoin lightning
```
Comment (dergigi) [inline data/blog/vitor-pamplona-receives-lts-grant.mdx:20]:
```suggestion
chats, custom feeds via [data vending machines](https://www.data-vending-machines.org/), and integrates 
```

File: data/blog/vitor-pamplona-receives-lts-grant.mdx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+"zaps" to enable users to exchange value directly within the app while
```
Comment (dergigi) [inline data/blog/vitor-pamplona-receives-lts-grant.mdx:21]: NIT: I'd end the sentence after "withing the app." -- and mention Tor support in a new sentence. (It's a very long sentence.)

**Review by dergigi — CHANGES_REQUESTED** [review]
Awesome! Love the quote. 💜

I have two minor suggestions, and let's not forget to update the date. Would be great if we could publish it today!

**Comment by Bayernatoor** [comment]
@Arvin21M Great post, so much content to talk about but it's well written and all the links work. I 2nd Gigi's comment regarding the image, it's awesome!

File: data/blog/vitor-pamplona-receives-lts-grant.mdx (hunk 3 above)
Comment (Bayernatoor) [reply data/blog/vitor-pamplona-receives-lts-grant.mdx:21]: I second this. It's a tad long. :)

Comment (Arvin21M) [reply data/blog/vitor-pamplona-receives-lts-grant.mdx]: Thank you guys 🙏 

Edited to the following:

> Privacy and security are further enhanced by allowing traffic routing through the Tor network via [Orbot](https://orbot.app/en/).

**Review by Arvin21M — COMMENTED** [review]
🚀

**Review by dergigi — APPROVED** [review]
Looks good to me ✅

**Comment by dergigi** [comment]
Live!
- https://opensats.org/blog/vitor-pamplona-receives-lts-grant
- [nostr post](https://njump.me/nevent1qqs8y9nye6d9fjhj36tr7m5xpst000ewasc5zh8kxwrlfcce74cenhqprpmhxue69uhhyetvv9ujumn0wdmksetjv5hxxmmdqyt8wumn8ghj7un9d3shjtnwdaehgu3wvfskueqpz3mhxue69uhhyetvv9ujuerpd46hxtnfdupzq7rn8p6hlsjavhxe9yu56hnhz085xcuw35jearw0t3emsd8ts50jr80gcg)

**Pull request merged**